otx/cli/tools/build.py

```diff
@@ -0,0 +1,60 @@
+"""Model templates searching tool."""
```

Comment:
Need to be update. It is description for find.py

Reply:
Added some clarification.